docs/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "doctocat-template",
+  "repository": "primer/doctocat-template",
```

We need to update the repository field to make the edit links work. 
```suggestion
  "repository": "primer/components",
```